modules/oadp-preparing-aws-credentials.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * rosa_backing_up_and_restoring_applications/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-preparing-aws-credentials_{context}"]
+= Preparing AWS credentials
+
+An AWS account must be ready to accept an OADP installation.
+
+.Procedure
+. Create the following environment variables by running the following command:
++
+[NOTE]
+====
+Change the cluster name to match your ROSA cluster, and ensure you are logged into the cluster as an administrator. Ensure that all fields are outputted correctly before moving on.
+====
++
+[source, terminal]
+----
+export CLUSTER_NAME=my-cluster <1>
```

Comment:
@weshayutin please advise

Reply:
This might be a good moment to add a wrapper script for them to avoid more extensive rework. The rule to place each command on a single line exists to facilitate the use of the copy button, which strips the leading "$."

Reply:
on a direct call with Wes we added in $ where necessary.